src/System.Private.CoreLib/src/Microsoft/Win32/SafeHandles/SafeWaitHandle.cs

```diff
@@ -36,7 +36,11 @@ public SafeWaitHandle(IntPtr existingHandle, bool ownsHandle) : base(ownsHandle)
 
         protected override bool ReleaseHandle()
         {
+#if PLATFORM_UNIX
```

Comment:
I thought the convention was to put Unix-specific code to SafeWaitHandle.Unix.cs.  Or do we allow single-line platform-specific #if's like the one here?